tensorflow/lite/tools/evaluation/utils.cc

```diff
@@ -135,6 +135,7 @@ TfLiteDelegatePtr CreateGPUDelegate() {
 #endif  // TFLITE_SUPPORTS_GPU_DELEGATE
 }
 
+#if defined(TFLITE_ENABLE_HEXAGON)
```

Comment:
I think this change will lead to some break for bazel-based builds where hexagon delegate is built based on platform-related macros? So, could you change cmake-related configurations (similar to the TFLITE_WITHOUT_XNNPACK macro) to achieve the same purpose?

just fyi, @terryheo @karimnosseir

Reply:
The build is actually broken currently when TFLITE_ENABLE_HEXAGON is **not** defined. I believe it is defined by default.

Reply:
@multiverse-tf Can you please assist on above comments from @NAgarwalla. Thanks!